packages/framework/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -16,47 +17,144 @@ describe("AzureClient", () => {
         },
     };
 
+    /**
+     * Scenario: test when Azure Client is instantiated correctly, it can create
+     * a container successfully.
+     *
+     * Expected behavior: an error should not be thrown nor should a rejected promise
+     * be returned.
+     */
     it("can create new Azure Fluid Relay container successfully", async () => {
-        const resources = client.createContainer(schema);
+        const azureContainer = client.createContainer(schema);
```

Comment:
nit: azureContainer isn't quite right because it's an anonymous object with the container and services.  i'd suggest going back to `resourcesP` or just `azureContainerP`